packages/calcite-components/src/components/radio-button-group/radio-button-group.tsx

```diff
@@ -6,24 +6,30 @@ import {
   h,
   Host,
   Listen,
+  Method,
   Prop,
+  State,
   VNode,
   Watch
 } from "@stencil/core";
 import { createObserver } from "../../utils/observers";
 import { Layout, Scale } from "../interfaces";
+import {
+  componentLoaded,
+  LoadableComponent,
+  setComponentLoaded,
+  setUpLoadableComponent
+} from "../../utils/loadable";
 
 /**
  * @slot - A slot for adding `calcite-radio-button`s.
  */
 @Component({
   tag: "calcite-radio-button-group",
   styleUrl: "radio-button-group.scss",
-  shadow: {
-    delegatesFocus: true
```

Comment:
Do we need to remove delegatesFocus for this component? If so can you summarize in the PR body? Just to document it since it seems like delegatesFocus is actually something we want to do more of.

Reply:
Added.